App/dfu/viewmodel/DfuViewModel.swift

```diff
@@ -97,7 +99,7 @@ class DfuViewModel : ObservableObject, DFUProgressDelegate, DFUServiceDelegate {
 
         let selectedFirmware = try! DFUFirmware(
```

Comment:
Does it compile?

Reply:
Yes. Forces a crash when it happens. But it was checked before, so for sure won't fail here.

Reply:
(Joke) Said no one ever... xD

Reply:
Do you have SwiftLint warnings? You might have no force unwrapping enforced, hence why Xcode doesn't build. (Just a guess.)